src/store/actions/featuredBanner.js

```diff
@@ -1,5 +1,10 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+// remember to rename actions as follows:
```

Comment:
i don't understand you forgot your note ?

Reply:
No I didn't forget it, actually this is note for me for a future modifications after finishing more urgent tasks..